ethereum/json-types/src/main/java/tech/pegasys/teku/ethereum/json/types/wrappers/ProposerDuty.java

```diff
@@ -11,14 +11,38 @@
  * specific language governing permissions and limitations under the License.
  */
 
-package tech.pegasys.teku.validator.api;
+package tech.pegasys.teku.ethereum.json.types.wrappers;
```

Comment:
given this comes from validator endpoints, id potentially make this scale neatly...
so if we had `json/types/validator`
maybe under that also builder.... because if we split out the builder and the type lives with the builder, then thats fairly neat, and the pojo can just stay clean...